src/System.Memory/ref/System.Memory.cs

```diff
@@ -538,6 +538,8 @@ public static partial class MemoryMarshal
         public static bool TryRead<T>(System.ReadOnlySpan<byte> source, out T value) where T : struct { throw null; }
         public static bool TryWrite<T>(System.Span<byte> destination, ref T value) where T : struct { throw null; }
         public static void Write<T>(System.Span<byte> destination, ref T value) where T : struct { }
+        public static ref T AsRef<T>(Span<byte> span) where T : struct { throw null; }
```

Comment:
nit: sort order, place after AsMemory (or auto-generate the ref).

Also, System.{ReadOnly}Span instead.